### ISSUE TITLE
Tesseract fails to recognize % sign in Hungarian language texts

### ISSUE THREAD

**Issue by Googulator** [tesseract-ocr/tessdata_best#40]
When the language is set to "hun" (Hungarian), Tesseract is unable to recognize the % sign. This sign is very commonly used in Hungarian to represent percentages, the same way as in English. Tesseract instead sees various letters and digits -  most commonly "96", sometimes "9", "69", "0", "S", "Z", or even nothing at all.

Even if I feed a generated image containing a % sign in large black type on a pure white background, I still can't get Tesseract to output the % sign, as long as the language is set to Hungarian.

Both the "fast" and "best" models suffer from this problem.

If I instead set the language to English, % sings are recognized without issue.

**Comment by stweil** [comment]
See [hun.unicharset](https://github.com/tesseract-ocr/langdata_lstm/blob/master/hun/hun.unicharset) which shows all known characters.

The percent sign was not part of the training data set, so Tesseract simply does not know that character with `hun`.

This can only be solved by new training, either from scratch or by fine tuning the existing incomplete model (which can add new characters).

An alternative would be using the `script/Latin` model which supports all western Europe languages.

**Comment by Googulator** [comment]
A model "which supports all western Europe languages" is not an option for Hungarian, because of ő and ű, which are not found in any Western European language.

**Comment by Shreeshrii** [comment]
Have you tried with `-l hun+eng`?

**Comment by stweil** [comment]
[Latin.unicharset](https://github.com/tesseract-ocr/langdata_lstm/blob/master/Latin.unicharset) includes both characters, so I suggest to try it. I updated my previous comment.

**Comment by Shreeshrii** [comment]
@stweil the more relevant unicharset will be the lstm-unicharset extracted from the script/Latin traineddata file. Latin.unicharset maybe a superset.

**Comment by stweil** [comment]
You are right, it is not identical, but that one also includes both characters.

**Comment by Googulator** [comment]
How do I test Latin.unicharset? Do I need to train a new model?

**Comment by stweil** [comment]
Just get `Latin.traineddata` and extract `Latin.unicharset` using `combine_tessdata`.
